Inventory/Items/Condom.gd

```diff
@@ -32,6 +32,11 @@ func getCondomBreakChance():
 	if(!OPTIONS.isContentEnabled(ContentType.RiskyCondoms)):
 		return 0.0
 	
+	# Check to make sure if PC is fucked with his own condoms still worked with the perk
+	var wearerIsNotPC: bool = getWearer() != GM.pc || (!isWornByWearer() && getWearer() == GM.pc)
+	if(wearerIsNotPC && GM.pc.hasPerk(Perk.FertilityDesireToBreed)):
+		return 90.0
```

Comment:
I'm not sure about this. What if I add a scene in the future where an npc fucks some other npc with a condom.. This would need to be handled differently then